python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import feelpp
+from feelpp.toolboxes.thermoelectric import *
+e=feelpp.Environment(sys.argv,opts=toolboxes_options("thermo-electric"))
+
+
+f=thermoelectric(dim=2,orderPotential=1,worldComm=e.worldCommPtr())
```

Comment:
heatfluid and modifiy arguments

Reply:
I will remove the code `heatfluid.py` it should not be there.